File: Teledyne DALSA/Sapera/Demos/NET/GigEFlatFieldDemo/CSharp/GigEFlatFieldDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace DALSA.SaperaLT.Demos.NET.CSharp.GigEFlatFieldDemo
{
    static class GigEFlatFieldDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            GigEFlatFieldDemoDlg dlg = new GigEFlatFieldDemoDlg();
            if(!dlg.IsDisposed)
                Application.Run(dlg);
        }
    }
}

File: Teledyne DALSA/Sapera/Demos/NET/GigEFlatFieldDemo/VB/GigEFlatFieldDemo.vb
Public Class GigEFlatFieldDemo
   'The main entry point for the application.
   <STAThread()> _
   Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Dim GigeFF As GigEFlatFieldDemoDlg = New GigEFlatFieldDemoDlg()
      If Not GigeFF.IsDisposed Then
         Application.Run(GigeFF)
      End If
   End Sub
End Class
